packages/tools/devtools/devtools-view/src/test/TelemetryView.test.tsx

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type DevtoolsFeatureFlags,
+	DevtoolsFeatures,
+} from "@fluidframework/devtools-core/internal";
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import { act } from "react-dom/test-utils";
+
+import { MessageRelayContext } from "../MessageRelayContext.js";
+import { TelemetryView } from "../components/index.js";
+
+import { assertNoAccessibilityViolations, MockMessageRelay } from "./utils/index.js";
+
+describe("TelemetryView Accessibility Check", () => {
+	const supportedFeatures: DevtoolsFeatureFlags = {
+		telemetry: true,
+		opLatencyTelemetry: true,
+	};
+	const mockMessageRelay = new MockMessageRelay(() => {
+		return {
+			type: DevtoolsFeatures.MessageType,
+			source: "TelemetryViewAccessibilityTest",
+			data: {
+				features: supportedFeatures,
+				devtoolsVersion: "1.0.0",
+				unsampledTelemetry: true,
+			},
+		};
+	});
+	it("TelemetryView is accessible", async () => {
+		const { container } = render(
+			<MessageRelayContext.Provider value={mockMessageRelay}>
+				<TelemetryView />{" "}
```

Comment:
That's not necessary, right?
```suggestion
				<TelemetryView />
```

Reply:
Oh must have been an artifact from linting. Will fix.